src/pages/api/public/events.ts

```diff
@@ -77,33 +77,32 @@ export default async function handler(
           ).id
         : obj.traceId;
 
-    // CHECK ACCESS SCOPE
-    const accessCheck = await checkApiAccessScope(authCheck.scope, [
-      ...(traceId ? [{ type: "trace" as const, id: traceId }] : []),
```

Comment:
Before we hat db-level assertion that observation and trace are in the same project; need to add it (adds latency) or ignore as we have projectId as kind of a tenant id now on the observations